dataconnect/dataconnect/connector/mutations.gql

```diff
@@ -0,0 +1,113 @@
+# Create a movie based on user input
+mutation createMovie(
+  $title: String!
+  $releaseYear: Int!
+  $genre: String!
+  $rating: Float
+  $description: String
+  $imageUrl: String!
+  $tags: [String!] = []
+) {
+  movie_insert(
+    data: {
+      title: $title
+      releaseYear: $releaseYear
+      genre: $genre
+      rating: $rating
+      description: $description
+      imageUrl: $imageUrl
+      tags: $tags
+    }
+  )
+}
+
+# Update movie information based on the provided ID
+mutation updateMovie(
+  $id: UUID!
+  $title: String
+  $releaseYear: Int
+  $genre: String
+  $rating: Float
+  $description: String
+  $imageUrl: String
+  $tags: [String!] = []
+) {
+  movie_update(
+    id: $id
+    data: {
+      title: $title
+      releaseYear: $releaseYear
+      genre: $genre
+      rating: $rating
+      description: $description
+      imageUrl: $imageUrl
+      tags: $tags
+    }
+  )
+}
+
+# Delete a movie by its ID
+mutation deleteMovie($id: UUID!) {
+  movie_delete(id: $id)
+}
+
+# Delete movies with a rating lower than the specified minimum rating
+mutation deleteUnpopularMovies($minRating: Float!) {
+  movie_deleteMany(where: { rating: { le: $minRating } })
+}
+
+# Add a movie to the user's watched list
+mutation addWatchedMovie($movieId: UUID!) @auth(level: USER) {
+  watchedMovie_upsert(data: { userId_expr: "auth.uid", movieId: $movieId })
+}
+
+# Remove a movie from the user's watched list
+mutation deleteWatchedMovie($userId: String!, $movieId: UUID!) @auth(level: USER) {
+  watchedMovie_delete(key: { userId: $userId, movieId: $movieId })
```

Comment:
Yeah, can you add a TODO to rewrite this once `key` support `userId_expr`?

Thanks a bunch. Missed this edge case when we implement server values.

There is a few similar cases below.